env/build_test.go

```diff
@@ -178,6 +143,8 @@ func testBuildEnv(t *testing.T, when spec.G, it spec.S) {
 				}
 			})
 			it("ignores case when initializing", func() {
+				platform.EXPECT().SupportsAssetPackages().Return(false)
```

Comment:
This is just a nit, but could we move it to the `Before` and change it to `true` (to be consistent with the "latest" platform). I just don't want to give the impression that somehow support (or not support) for asset packages is related to Windows.